src/traces/mesh3d/attributes.js

```diff
@@ -57,11 +87,28 @@ module.exports = {
         role: 'style',
         dflt: -1,
         description: [
-            'Sets the shape of the mesh',
-            'If *-1*, Delaunay triangulation is used',
-            'If *>0*, the alpha-shape algorithm is used',
-            'If *0*,  the convex-hull algorithm is used',
-            'An alternative to the `i`, `j`, `k` indices triplets.'
+            'Determines how the mesh surface triangles are derived from the set of',
+            'vertices (points) represented by the `x`, `y` and `z` arrays, if',
+            'the `i`, `j`, `k` arrays are not supplied.',
+            'For general use of `mesh3d` it is preferred that `i`, `j`, `k` are',
+            'supplied, because calculating the surface from points can be very',
+            'time consuming and usually lead to artifacts.',
+
+            'If *-1*, Delaunay triangulation is used, which is mainly suitable if the',
+            'mesh is a single, more or less layer surface that is perpendicular to `delaunayaxis`.',
+            'In case the `delaunayaxis` intersects the mesh surface at more than one point',
+            '(e.g. sphere, cube, potato, animal, donut etc. shapes) it will lead to triangles',
+            'that are very long in the dimension of `delaunayaxis`.',
+
+            'If *>0*, the alpha-shape algorithm is used. In this case, the `alphahull` value',
```

Comment:
🎉 thanks! exactly the info I was looking for.
To be a bit more concise I would say:
'If *>0*, the alpha-shape algorithm is used. In this case, the `alphahull` value signals the use of the alpha-shape algorithm, *and* its value acts as the parameter for the mesh fitting.